sdk/reservations/Microsoft.Azure.Management.Reservations/src/Generated/AzureReservationAPIClient.cs

```diff
@@ -419,7 +425,7 @@ public async Task<AzureOperationResponse<IList<Catalog>>> GetCatalogWithHttpMess
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "reservedResourceType");
             }
-            string apiVersion = "2019-04-01";
+            string apiVersion = "2020-10-01-preview";
```

Comment:
I notice that this api is also included in the stable tag in your readme.md in the azure-rest-api-specs repo.  This should not have been allowed by the PR reviewer.  Only stable APIs should be included in stable tag versions.  This will likely cause iossues with other SDKs.